src/Compilers/Metadata/CustomDebugInfoReader.cs

```diff
@@ -353,8 +353,14 @@ public static ImmutableArray<ImmutableArray<string>> GetCSharpGroupedImportStrin
                         }
 
                         int moduleInfoMethodToken = DecodeForwardToModuleRecord(record.Data);
-                        ImmutableArray<string> allModuleInfoImportStrings = reader.GetMethodByVersion(moduleInfoMethodToken, methodVersion).GetImportStrings();
-                        ArrayBuilder<string> externAliasBuilder = ArrayBuilder<string>.GetInstance();
+
+                        ImmutableArray<string> allModuleInfoImportStrings = getMethodImportStrings(moduleInfoMethodToken, arg);
+                        if (allModuleInfoImportStrings.IsDefault)
+                        {
+                            throw new InvalidOperationException(string.Format("Forwarded to non-existing method {0}", FormatMethodToken(moduleInfoMethodToken)));
```

Comment:
Actually, GetImportStrings() checked for null receiver and returned return ImmutableArray<string>.Empty; :(
OK, I'll remove throwing.